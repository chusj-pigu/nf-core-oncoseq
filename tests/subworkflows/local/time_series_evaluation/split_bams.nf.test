nextflow_workflow {

    name "Test Workflow SPLIT_BAMS_TIME"
    script "subworkflows/local/time_series_evaluation/split_bams.nf"
    workflow "SPLIT_BAMS_TIME"

    test("Test Workflow SPLIT_BAMS_TIME") {

        when {
            params {
                max_cpus = 2
            }
            workflow {
                """
                // define inputs of the workflow here. Example:
                input[0] = tuple(
                [id: "test_sample"],
                file("${projectDir}/test_data/alignments/sample1_mito_align.bam")
                )
                input[1] = tuple 0, 4
                """
            }
        }

        then {
            assert workflow.success
            assert snapshot(workflow.out).match()
        }

    }
}
