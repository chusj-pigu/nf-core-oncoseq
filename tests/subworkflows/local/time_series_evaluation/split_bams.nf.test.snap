{
    "Test Workflow SPLIT_BAMS_TIME": {
        "content": [
            {
                "0": [
                    [
                        {
                            "id": "test_sample"
                        },
                        "test_sample-time-0h-4h.bam:md5,58651f57dee5533e8bb3f3298bb23c0e"
                    ]
                ],
                "1": [
                    
                ],
                "ch_bam_out": [
                    [
                        {
                            "id": "test_sample"
                        },
                        "test_sample-time-0h-4h.bam:md5,58651f57dee5533e8bb3f3298bb23c0e"
                    ]
                ],
                "ch_versions": [
                    
                ]
            }
        ],
        "meta": {
            "nf-test": "0.9.2",
            "nextflow": "24.10.4"
        },
        "timestamp": "2025-04-11T20:31:13.233718676"
    }
}